README.md

```diff
@@ -1,8 +1,128 @@
-# Getting Started with Create React App
+<div align="center">
+  <h1><b>Book-Store</b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Getting Started with Create React App](#getting-started-React)
+  - [Available Scripts](#available-scripts)
+  - [Learn More](#learn-more)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 book-store <a name="about-project"></a>
+
+**book-store**  It is a Single Page App (SPA) that allows users to:
+- add books by it author name and book title.
+- Remove books
+to build this application, we use React.
+
+# 👁‍🗨 Preview
+
+Pleaseholder
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML5 / CSS / JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>IDE</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Hosting</summary>
+  <ul>
+    <li><a href="https://github.com/">GitHub</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Library</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Make simple calculations**
+- **Read a random math-related quote**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://github.com/atatm/book-store.git)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+- Get a copy of the project
+- Come into the project directory
+- Open the project using code editor of your choice
+- Open your prefered browser
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project:
+- You need a GitHub account.
+- You need to have a Git client installed on your computer.
+- You need to have a copy of the HTML CSS website codebase on your computer.
+- You need a code Editor of your choice.
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/atatm/book-store.git
+```
+
```

Comment:
### Your Summary 🔴
![invalid-summary](https://user-images.githubusercontent.com/93039359/209281665-43c3429f-b864-41dc-af64-8101dc08cf0e.PNG)

### Descriptive Summary 🟢
![summary](https://user-images.githubusercontent.com/93039359/209281681-ebf349b3-feb0-40be-a00f-9ba4dbf1c1b2.PNG)

- [ ] Your summary is not descriptive enough. Because no one will be able to understand what type of api you are connecting with your application and why you are connecting it with your app. So please make it descriptive enough as i have shown in the second image above 👍